docs/src/guides/setup-dev.md

```diff
@@ -77,84 +30,27 @@ Rosetta may cause problems that are hard to spot and debug, so make sure to chec
 
 If you are a NixOS user or would like to have a reproducible environment, skip to the section about `nix`.
 
-## Docker
+### Basic packages
 
-Install `docker`. It is recommended to follow the instructions from the
-[official site](https://docs.docker.com/install/).
-
-Note: currently official site proposes using Docker Desktop for Linux, which is a GUI tool with plenty of quirks. If you
-want to only have CLI tool, you need the `docker-ce` package and you can follow
-[this guide](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-20-04) for Ubuntu.
-
-Installing `docker` via `snap` or from the default repository can cause troubles.
-
-You need to install both `docker` and `docker compose`.
-
-**Note:** `docker compose` is installed automatically with `Docker Desktop`.
-
-**Note:** On linux you may encounter the following error when you’ll try to work with `zksync`:
-
-```
-ERROR: Couldn't connect to Docker daemon - you might need to run `docker-machine start default`.
-```
-
-If so, you **do not need** to install `docker-machine`. Most probably, it means that your user is not added to
-the`docker` group. You can check it as follows:
-
-```bash
-docker-compose up # Should raise the same error.
-sudo docker-compose up # Should start doing things.
-```
-
-If the first command fails, but the second succeeds, then you need to add your user to the `docker` group:
-
-```bash
-sudo usermod -a -G docker your_user_name
-```
-
-After that, you should logout and login again (user groups are refreshed after the login). The problem should be solved
-at this step.
-
-If logging out does not resolve the issue, restarting the computer should.
-
-## Node.js & Yarn
-
-1. Install `Node` (requires version `v20`). The recommended way is via [nvm](https://github.com/nvm-sh/nvm).
-2. Install `yarn`. Can be done via `npm install -g yarn`. Make sure to get version 1.22.19 - you can change the version
-   by running `yarn set version 1.22.19`.
-
-## clang
-
-In order to compile RocksDB, you must have LLVM available. On debian-based linux it can be installed as follows:
+Install the basic packages:
 
 On debian-based linux:
 
 ```bash
-sudo apt-get install build-essential pkg-config cmake clang lldb lld
+sudo apt install git build-essential pkg-config cmake clang lldb lld libssl-dev libpq-dev apt-transport-https ca-certificates curl software-properties-common
 ```
 
-On macOS:
-
-You need to have an up-to-date `Xcode`. You can install it directly from `App Store`. With Xcode command line tools, you
-get the Clang compiler installed by default. Thus, having XCode you don't need to install `clang`.
-
-## OpenSSL
-
-Install OpenSSL:
-
 On mac:
 
 ```bash
-brew install openssl
+brew install git build-essential pkg-config cmake clang lldb lld libssl-dev libpq-dev apt-transport-https ca-certificates curl software-properties-common
```

Comment:
Are these needed? Have you tested it?

Reply:
These are the same we had before, I only added curl.
I don't know if they are all necessary, and it would be hard to check that. For example, libssl-dev is only used to compile RocksDB, so I would only figure that out by compiling the node. Unless I run all commands on ZK Stack CLI, I don't know how I would check if the dependencies are all used.

Reply:
It's a command for `brew` (MacOS). Current `setup-dev` only has two commands for homebrew: `openssl` and `libpq` (the latter is actually not required.

Reply:
Fairly so.